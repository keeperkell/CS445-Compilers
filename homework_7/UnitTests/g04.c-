## C-S22
int z;

int cat(int z, x)
 begin 
    return z * x - x;
 end 

main()
 begin 
    output(cat(33, 911));
 end 
