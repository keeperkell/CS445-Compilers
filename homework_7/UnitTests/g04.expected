WARNING(2): The variable 'z' seems not to be used.
Number of warnings: 1
Number of errors: 0
------------------------------------
Loading file: UnitTests/g04.tm
29152
Bye.
